pwa-devdocs/src/venia-pwa-concept/setup/index.md

```diff
@@ -90,21 +90,29 @@ Don't forget to install the [Venia sample data][]!
 
 Before you run the server, generate build artifacts for Venia using the following command in the **project root directory**:
 
-`yarn run build`
+``` sh
+yarn run build
+```
 
 ### Run the server
 
 Use any of the following commands from the **project root directory** to start the server:
 
-`yarn run watch:venia`
+``` sh
+yarn run watch:venia
+```
 
 : Starts the Venia storefront development environment.
 
-`yarn run watch:all`
+``` sh
+yarn run watch:all
+```
 
 : Runs the full PWA Studio developer experience, which include Venia hot-reloading and concurrent Buildpack/Peregrine rebuilds.
 
-`yarn run build && yarn run stage:venia`
+``` sh
+yarn run build && yarn run stage:venia
+```
```

Comment:
Changing this line from an inline code format to a block code format messes up the definition list format when the HTML page is generated. Suggest removing this change.